lib/domain.gi

```diff
@@ -205,6 +205,18 @@ InstallMethod( GeneratorsOfDomain,
     AsList );
 
 
+#############################################################################
+##
+##  PrintObj
+##
+InstallMethod( PrintObj,
+    "for a domain with HasGeneratorsOfDomain",
```

Comment:
```suggestion
    "for a domain with GeneratorsOfDomain",
```
would be my preference, but it's not really that important.

Reply:
updated :+1: